Sources/Workspace/UserToolchain.swift

```diff
@@ -78,6 +78,9 @@ public final class UserToolchain: Toolchain {
     /// Search paths from the PATH environment variable.
     let envSearchPaths: [AbsolutePath]
 
+    /// Only use search paths, do not fall back to `xcrun`.
+    let useXcrun: Bool
```

Comment:
used anywhere?

Reply:
Yah, `getClangCompiler()` and `getLLDB()`.

Reply:
oh I see now, its an argument for the static func, and we use the instance variable for the instance ones 👍